app/subscriber/src/components/section/Section.tsx

```diff
@@ -65,7 +65,23 @@ export const Section: React.FC<ISectionProps> = ({
           </div>
         )}
       </div>
-      {open && <div className="section-body">{children}</div>}
+      {open && (
```

Comment:
This is required if we don't want to render children that don't need to be rendered.